cli/cmd/list_cmd.go

```diff
@@ -0,0 +1,48 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/kubeshop/tracetest/cli/analytics"
+	"github.com/spf13/cobra"
+	"go.uber.org/zap"
+)
+
+var listCmd = &cobra.Command{
+	Use:    "list [resource type]",
+	Long:   "List resources from your Tracetest server",
+	Short:  "List resources",
+	PreRun: setupCommand(),
+	Args:   cobra.MinimumNArgs(1),
```

Comment:
To think in the future: should we add optional parameters to sort outputs and filter results?

Reply:
I think those can be handled using flags, that will be part of the next pr when introducing polling profiles